test/functional/neutronless/conftest.py

```diff
@@ -39,6 +239,14 @@ def fin():
     return bigip
 
 
+def debug_msg(status):
```

Comment:
Note that this is quite handy... upon a failed test, it will output the debug statement into the STDOUT with the line from the test that called it; thus, you know when/where the deployed state was performed.